docs/sdk-docs/installing-sdk/README.md

```diff
@@ -0,0 +1,24 @@
+# Install and Import the SDK
+
+You can install the Raiden SDK as a node module by running:
+
+```bash
+npm install raiden-ts
+```
+
+After installing the SDK you can simply import Raiden to your JavaScript or TypeScript project.
+
+```typescript
+import { Raiden } from 'raiden-ts';
+
+# async factory
```

Comment:
`#` is not valid JavaScript

Reply:
Yes, we have a few parts with python comments also in the rest of the docs. It' is not only here.
All these code snippets should be updated. We should create an issue about that.